orchagent/aclorch.cpp

```diff
@@ -2837,9 +2840,16 @@ bool AclOrch::removeAclTable(string table_id)
     if (deleteUnbindAclTable(table_oid) == SAI_STATUS_SUCCESS)
     {
         auto stage = m_AclTables[table_oid].stage;
+        auto type = m_AclTables[table_oid].type;
+
         sai_acl_stage_t sai_stage = (stage == ACL_STAGE_INGRESS) ? SAI_ACL_STAGE_INGRESS : SAI_ACL_STAGE_EGRESS;
         gCrmOrch->decCrmAclUsedCounter(CrmResourceType::CRM_ACL_TABLE, sai_stage, SAI_ACL_BIND_POINT_TYPE_PORT, table_oid);
 
+        if (type != ACL_TABLE_PFCWD)
```

Comment:
If another ACL table is added similar to PFCWD (which doesn't bind to LAG), then this leaves a gap. But I don't think there is a better way other than having a logic to go over the bindpoint list of acl table and address it in a generic way. So may be for a future commit.

Reply:
@prsunny If there is not a generic way to address the issue, why we need this PR in aclorch?

Reply:
@wangshengjun , this PR is definitely addressing the issue. I was referring to a different approach to handle this so that crm can be incremented based on bind list rather than duplicating the code at multiple places. Like a slight re-design.

Reply:
@prsunny 
Scenario 1: only port bindings for ACL tables
With this PR
CRM: port + 1, lag + 1 (wrong)
Without this PR
CRM:port + 1 (right)
Scenario 2: only lag bindings for ACL tables
With this PR
CRM: port + 1, lag + 1 (wrong)
Without this PR
CRM:port + 1  (wrong)
Scenario 3: both port and lag bindings for ACL tables
With this PR
CRM: port + 1, lag + 1 (right)
Without this PR
CRM:port + 1  (wrong)

Reply:
@wangshengjun please take a look at: https://github.com/Azure/sonic-swss/blob/04e7341cb2f45141f8bb33cd444922f3dcfd1952/orchagent/aclorch.cpp#L1324

All ACL tables have port + LAG bindings regardless of what interfaces you add to them. Whether that is the correct behavior or not is beyond the scope of this PR.

@prsunny I agree with you that this is not the most robust way of handling this, but I think the overall create/delete flow for ACL tables is pretty fragmented and needs to be revisited. I think we should take this up in a future PR.